### DISCUSSION TITLE
[`nerdctl build`] Why can't nerdctl pull images in dockerfile (`x509: cannot validate certificate for xxxx because it doesn't contain any IP SANs`)

### DISCUSSION THREAD

**Discussion by l1douhua** [containerd/nerdctl#1362] [Q&A]
The problem is that if I use it directly in the pod, I can pull the image directly, but not in the Dockerfile

nerdctl pull --insecure-registry xxxx:30003/zikayou/serverjre:v1
time="2022-09-06T11:00:27Z" level=warning msg="skipping verifying HTTPS certs for "xxxx:30003""
xxxx:30003/zikayou/serverjre:v1: resolving |�[32m�[0m--------------------------------------|
elapsed: 0.1 s total: 0.0 B (0.0 B/s)
xxxx:30003/zikayou/serverjre:v1: resolved |�[32m++++++++++++++++++++++++++++++++++++++�[0m|
manifest-sha256:735c9e6406bc633a27735f93ae23b61b5a801d66010a3cce570938ca2331cc88: waiting |�[32m�[0m--------------------------------------|
elapsed: 0.2 s total: 0.0 B (0.0 B/s)

nerdctl build --insecure-registry -t xxxx:30003/zikayou/springboot-demo:v0.0.4 .

Dockerfile:1

1 | >>> FROM xxxx:30003/zikayou/serverjre:v1
2 | RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
3 | ADD target/springboot-demo-0.0.1-SNAPSHOT.jar /springboot-demo-0.0.1-SNAPSHOT.jar

error: failed to solve: xxxx:30003/zikayou/serverjre:v1: failed to do request: Head "https://xxxx:30003/v2/zikayou/serverjre/manifests/v1": x509: cannot validate certificate for xxxx because it doesn't contain any IP SANs
time="2022-09-06T11:00:36Z" level=fatal msg="unrecognized image format"

**Comment by fahedouch** [comment]
switched to issue https://github.com/containerd/nerdctl/issues/1363
